src/main/java/net/mcreator/blockly/BlocklyBlockUtil.java

```diff
@@ -118,4 +119,49 @@ public static Color getBlockColorFromHUE(int hue) {
 		return Color.getHSBColor(hue / 360f, 0.37f, 0.6f);
 	}
 
+	/**
+	 * Calculate the HUE of a block from RGB
+	 *
+	 * @param var The variable element to use
+	 * @return Built-in color or HUE for other blocks
+	 */
+	public static String getHUEFromRGB(VariableElementType var) {

```

Comment:
Name of this methodis misleading, it is not from rgb but from variable type. It also returns blockly color, not hue in all cases